sdk/src/main/kotlin/io/rover/sdk/ui/blocks/poll/image/ViewImagePoll.kt

```diff
@@ -77,7 +79,7 @@ internal class ViewImagePoll(override val view: LinearLayout) :
             bindQuestion(viewModel.imagePoll)
             setupOptionViews(viewModel, imageLength)
 
-            viewModel.multiImageUpdates.subscribe(
+            viewModel.multiImageUpdates.first().subscribe(
```

Comment:
if you follow through on above comment, you would want to undo this bit here for sure :)